godot-kotlin/godot-library/src/nativePublic/kotlin/godot/annotation/RegisterAnnotation.kt

```diff
@@ -14,7 +14,7 @@ annotation class RegisterClass(val isTool: Boolean = false)
 @Target(AnnotationTarget.PROPERTY)
 @Retention(AnnotationRetention.RUNTIME)
 annotation class RegisterProperty(
-    val visibleInEditor: Boolean = false,
+    val visibleInEditor: Boolean = true,
```

Comment:
Why default value is true ?

Reply:
Most properties you want to register you also want to be visible in the editor.
You only don't want it to be visible for accessign through gdScript. here we took a kotlin first approach. We were expecting most users who use this binding to just use kotlin and to use other languages lesser. so most of the times the user would have to provide a boolean.

Reply:
I think it's mainly a question how we want to present the binding. as a additional way to programming for godot. or the main way for programming.
Personally I'm for the last option

Reply:
I think most users will use this binding to create games, so `visibleInEditor = true`.

Reply:
I agree with you, if performances permit it :)
So LGTM.